server/src/main/java/org/cloudfoundry/identity/uaa/logging/LogSanitizerUtil.java

```diff
@@ -1,10 +1,15 @@
 package org.cloudfoundry.identity.uaa.logging;
 
+import javax.annotation.Nullable;
```

Comment:
Please check other parts uses org.springframework.lang.Nullable;

In general I would omit the annotation

Reply:
change import to org.springframework.lang.Nullable, which is used in IdentityProviderEndpoints.java and IdpAliasFailedException.java